src/udev/scsi_id/scsi_serial.c

```diff
@@ -361,14 +468,37 @@ static int scsi_inquiry(struct scsi_id_device *dev_scsi, int fd,
 }
 
 /* Get list of supported EVPD pages */
-static int do_scsi_page0_inquiry(struct scsi_id_device *dev_scsi, int fd,
-                                 unsigned char *buffer, unsigned len) {
+static int sgio_do_scsi_page0_inquiry(struct scsi_id_device *dev_scsi, int fd,
+                                      unsigned char *buffer, unsigned len) {
+        int retval;
+
+        memzero(buffer, len);
+        retval = sgio_scsi_inquiry(dev_scsi, fd, 1, 0x0, buffer, len);
+        if (retval < 0)
+                return 1;
```

Comment:
Hmmm. That violates our coding style, but mimic the existing code below. So, let's keep this now.

Reply:
Done